src/fragments/lib/logging/ios/setup_logging/20_initialize_with_config.mdx

```diff
@@ -0,0 +1,43 @@
+In your mobile app, create and add a `amplifyconfiguration_logging.json` to the same location as the `amplifyconfiguration.json` file.  Ensure that the file is included in the `Copy Bundle Resources` build phase.
+
+The `<log-group-name>` and `<region>` is the value you specified in the CDK construct as part of provisioning your backend resources.  These values can also be found at the end of the output logs when deploying the sample CDK construct.  The configuration file is the data source for the logging plugin to know where, when and what logs to sends.  The example below configures the logging plugin to automatically send all logs at log level `ERROR` at 60 seconds interval and store logs locally up to 1MB.
+
+```json
+{
+    "awsCloudWatchLoggingPlugin": {
+        "enable": true,
+        "logGroupName": "<log-group-name>",
+        "region": "<region>",
+        "localStoreMaxSizeInMB": 1,
+        "flushIntervalInSeconds": 60,
+        "loggingConstraints": {
+            "defaultLogLevel": "ERROR"
+        }
+    }
+}
+```
+
+To use the Amplify Logger and Amplify Auth categories in your app, you need to create and configure their corresponding plugins by calling the `Amplify.add(plugin:)` and `Amplify.configure()` methods.
+
+Add the following imports to the top of your main `App` file:
+
+```swift
+import Amplify
+import AWSCognitoAuthPlugin
+import AWSCloudWatchLoggingPlugin
+```
+
```

Comment:
add the following code to its initializer?

What does that mean? should we say Add the following code to your app's initializer?

Reply:
"its" refers to "your main `App` file" that is mentioned in line 22. As in "Add the following code to the initializer of the same file in which you added the previous imports".

I think saying "your app's initializer" is also correct, but it also kinda loses the context of the previous paragraph and someone might think it's not the same file.

Reply:
We use the same verbiage in all other existing code snippets in other categories, so I'd like to keep it consistent.